src/components/ContentWrapper.jsx

```diff
@@ -280,18 +302,25 @@ const ContentWrapper = (props) => {
   const menuItemRender = ({
     path, icon, name, disableIfNoExperiment, disabledByPipelineStatus,
   }) => {
-    const noExperimentDisable = !experimentId ? disableIfNoExperiment : false;
+    const pipelinesCompleted = () => {
+      if (pipelineStatusKey === pipelineStatus.SUCCEEDED
+        && gem2sStatusKey === pipelineStatus.SUCCEEDED) {
+        return true;
+      }
+      return false;
+    };
+    const notProcessedExperimentDisable = routeExperimentId || pipelinesCompleted() ? false : disableIfNoExperiment;
```

Comment:
I think this is confusing. Instead of doing this, maybe you can do
```
const notProcessedExperimentDisable = routeExperimentId || pipelinesCompleted()
```

And then on line inside.`Menu.Item` `disabled` on line 323, you just do 
```
...
disabled={  notProcessedExperimentDisable || noExperimentDisable || pipelineStatusDisable }
...
```
With short-circuiting, this should be the same right?